stix2elevator/convert_stix.py

```diff
@@ -1357,7 +1358,7 @@ def handle_missing_properties_of_ciq_instance(identity_instance, ciq):
         fill_in_extension_properties(identity_instance, container, extension_definition_id)
 
 
-def convert_identity(identity, env, created_by_ref_source, parent_id=None, temp_marking_id=None):
+def convert_identity(identity, env, created_by_ref_source, parent_id=None, temp_marking_id=None, victim=False):
```

Comment:
Can you explain this new `victim` parameter? Why is `name` not required if it's a victim?

Reply:
In STIX 1.x victim targeting is specified via a TTP.  So within processing for a TTP, convert_identity is called.  Because a ttp can also have a title, we don't want convert_identity to make the final decision of needing to generate the placeholder when it is handing victims.